gui_app.py

```diff
@@ -609,6 +750,23 @@ def update_app_button(self, evt=None):
         self.gui.app_button.state(['!disabled'])
         self.gui.app_buttonvar.set(f"Run {config.FLPRODUCT}")
 
+    def update_latest_appimage_button(self, evt=None):
+        status, reason = utils.compare_recommended_appimage_version()
+        msg = None
+        if status == 0:
+            state = '!disabled'
+        elif status == 1:
+            state = 'disabled'
+            msg = "This button is disabled. The AppImage is already set to the latest recommended."  # noqa: E501
+        elif status == 2:
+            state = 'disabled'
+            msg = "This button is disabled. The AppImage version is newer than the latest recommended."  # noqa: E501
+        if msg:
+            gui.ToolTip(self.gui.latest_appimage_button, msg)
+        self.clear_message_text()
+        self.stop_indeterminate_progress()
+        self.gui.latest_appimage_button.state([state])
+
```

Comment:
By moving this code here rather than checking it during the creation of the AppImage, the button remains "Enabled" but now on hover says creates a popup saying it is disabled.

Is TK able to handle a dynamic change like this? Can this change be moved to gui_app.py, line 566?